### DISCUSSION TITLE
NeoCoolCam not reconized (NAS PD01)

### DISCUSSION THREAD

**Discussion by Touma02** [zwave-js/zwave-js-ui#3083] [General]
Hello Guys,

Is it possible to put the Neo Cool cam NAS PD 01 with this informations :
#
![zwave1](https://github.com/zwave-js/zwave-js-ui/assets/133054353/8ed97872-24d5-4e8e-bf58-4a44629d8b7e)
![zwave2](https://github.com/zwave-js/zwave-js-ui/assets/133054353/6df31c6f-8c89-43e9-aafe-5f0a4dc9691b)

thank you so much

**Comment by robertsLando** [comment]
Sorry what?

**Reply by Touma02** [reply]
I bought a NAS PD01 but it is ont recognized. I think is not thé same version of NAS PD01 already ok on zwavejs. Can you add it ?

**Reply by robertsLando** [reply]
Open an issue: https://github.com/zwave-js/node-zwave-js/issues/new/choose

Choose 'Request missing device configuration'
